addons/l10n_id_efaktur_coretax/models/account_move_line.py

```diff
@@ -21,14 +21,18 @@ def _l10n_id_coretax_build_invoice_line_vals(self, vals):
         luxury_tax = self.tax_ids.filtered(lambda tax: tax.tax_group_id == luxury_tax_group)
         regular_tax = self.tax_ids - luxury_tax
 
+        # "Price" is unit price calculation excluding tax and discount
+        # "TotalDiscount" is total of "Price" * quantity * discount
+        tax_res = self.tax_ids.compute_all(self.price_unit, quantity=1, currency=self.currency_id, product=self.product_id, partner=self.partner_id, is_refund=self.is_refund)
```

Comment:
Try to use `self._prepare_invoice_aggregated_taxes()` instead

Reply:
Oups we're on the line.

I guess `_convert_to_tax_base_line_dict()` and `_compute_taxes_for_single_line` would be a better solution. :bowing_man:

Reply:
yeah I think _conver_to_tax_base_line_dict and _compute_taxes_for_single_line should work here